rocketpy/Flight.py

```diff
@@ -1563,6 +1575,258 @@ def uDot(self, t, u, postProcessing=False):
 
         return uDot
 
+    def uDotGeneralized(self, t, u, postProcessing=False):
+        """Calculates derivative of u state vector with respect to time when the
+        rocket is flying in 6 DOF motion in space and significant mass variation
+        effects exist. Typical flight phases include powered ascent after launch
+        rail.
+
+        Parameters
+        ----------
+        t : float
+            Time in seconds
+        u : list
+            State vector defined by u = [x, y, z, vx, vy, vz, q0, q1,
+            q2, q3, omega1, omega2, omega3].
+        postProcessing : bool, optional
```

Comment:
Not related to this PR but... When are we going to depracate `postProcessing` argument?

Reply:
I guess the v1 release is the best moment to do so. We can also remove all the other deprecated functions

Reply:
For future reference: 
- https://peps.python.org/pep-0387/
- https://peps.python.org/pep-0702/#deprecation-policy